src/SMG/ManagerBundle/Controller/ManagersController.php

```diff
@@ -128,4 +126,18 @@ public function patchUserDisableAction(User $user) {
 
         return $this->handleView(new View());
     }
+
+    /**
+     * @param User $user
+     */
+    public function patchUserEnableAction(User $user)
+    {
+        $this->throwIfClientNot('backend');
+
+        $user->setEnabled(true);
+
+        $this->get('fos_user.user_manager')->updateUser($user);
+
+        return $this->handleView(new View());
```

Comment:
can you try if  no return works (i think so)  (same for the other methods who are supposed to generate a 204)